### PR TITLE
fix: Changed the Attendee and Ticket Entity from DBFlow to Room

### LINKED ISSUE #1348: Replace the current database with Room
### Chore

**Description**
We should start using Room database (AAC) in the project.

**Would you like to work on the issue?**
No

### PR DESCRIPTION
**Pull request by tavishjain**
Partially Fixes #1348 

Checklist:

- [X] I have checked for PMD and check-style issues 
- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream development branch.

Changes: [Add here what changes were made in this issue and if possible provide links.]

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.EqualsAndHashCode;
 import lombok.experimental.Delegate;
 
-@Data
-@Builder
-@Type("attendee")
-@AllArgsConstructor
```
Comment (ShridharGoel) [inline app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: Why removing these annotations?

File: app/src/main/java/com/eventyay/organizer/data/db/configuration/OrgaDatabase.java [hunk 2]
```diff
@@ -84,7 +79,7 @@ public MigrationTo5(Class<Attendee> table) {
         @Override
         public void onPreMigrate() {
             super.onPreMigrate();
-            addColumn(SQLiteType.INTEGER, "checking");
+            addColumn(SQLiteType.INTEGER, "checking");a
```
Comment (ShridharGoel) [inline app/src/main/java/com/eventyay/organizer/data/db/configuration/OrgaDatabase.java]: Typo

Comment (tavishjain) [reply app/src/main/java/com/eventyay/organizer/data/db/configuration/OrgaDatabase.java]: Fixed

File: app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java [hunk 3]
```diff
@@ -1,39 +1,35 @@
...
 import lombok.Data;
-import lombok.EqualsAndHashCode;
 import lombok.experimental.Delegate;
 
 @Data
 @Builder
 @Type("attendee")
 @AllArgsConstructor
-@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
-@EqualsAndHashCode(callSuper = false, exclude = { "attendeeDelegate", "checking" })
```
Comment (ShridharGoel) [inline app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: These two are also Lombok annotations which shouldn't be removed

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/data/attendee/Attendee.java]: JsonNaming is Jackson annotation and EqualsAndHashCode is Lombok annotation

**Pull request closed without merging**